src/ComposableController.test.ts

```diff
@@ -169,7 +173,7 @@ describe('ComposableController', () => {
           ipfsGateway: 'https://ipfs.io/ipfs/',
           lostIdentities: {},
           selectedAddress: '',
-          useStaticTokenList: false,
+          useTokenDetection: false,
```

Comment:
I know this is just the test file, does it make sense to have token detection ON by default?